pkg/updater/updater.go

```diff
@@ -214,7 +242,9 @@ func (u *updaterImpl) BootstrapVersion(ctx context.Context, pkg string, version
 }
 
 // BootstrapURL installs the stable version of the package.
-func (u *updaterImpl) BootstrapURL(ctx context.Context, url string) error {
+func (u *updaterImpl) BootstrapURL(ctx context.Context, url string) (err error) {
+	span, ctx := tracer.StartSpanFromContext(ctx, "BootstrapURL")
```

Comment:
operation names are used in datadog metrics and resource as tags. By default operation name = resource
the metric is case sensitive and the tag is converted to a lower case

for nice interaction in the app, the easiest is to follow tags normalisation: https://docs.datadoghq.com/getting_started/tagging/?s=metric#define-tags